webpack.development.js

```diff
@@ -1,19 +1,21 @@
 const config = require('./webpack.config');
 const webpack = require('webpack');
 
-config.devtool = 'eval-cheap-module-source-map';
-config.externals = ['react'];
-config.plugins.push(
-  new webpack.optimize.LimitChunkCountPlugin({
-    maxChunks: 1
-  })
-);
+module.exports = (env) => {
```

Comment:
can we pull this bundle analyzer refactor into a separate PR that we can merge first?